langlib/lang.boolean/src/main/ballerina/boolean.bal

```diff
@@ -24,6 +24,11 @@ import ballerina/jballerina.java;
 # and also `1` for true and `0` for `false`.
 # This is the inverse of function ``value:toString`` applied to a `boolean`.
 #
+# ```ballerina
+# string str = "true";
+# boolean booleanValue = check boolean:fromString(str);
```

Comment:
```suggestion
# string str = "true";
# boolean bool = check boolean:fromString(str);
```

Since we are using `str` for the string. 

Alternatively, we can probably use some kind of a real-world scenario

```suggestion
# string enabledStr = "true";
# boolean enabled = check boolean:fromString(enabledStr);
```

Reply:
Some other examples with the combination of lower and upper cases, as well as `"1"`, and `"0"` also could be used.
```
string trueValue = "True";
boolean trueFlag = check boolean:fromString(trueValue);

string falseDigitStr = "0";
boolean falseFlag = check boolean:fromString(falseDigitStr);
```

Reply:
Yes. But it will increase the number of examples in here ryt?
Please refer to https://github.com/ballerina-platform/ballerina-lang/pull/38870#discussion_r1034633396